src/styles/_typography.scss

```diff
@@ -0,0 +1,76 @@
+$fontDir: '../fonts/';
```

Comment:
Just curious, do we really need all these fonts?

Reply:
Copy-paste from fingerprint.com but I would say yes. It's just two fonts (one for text, one for code) in two formats (the old one included for backward compatibility I assume) at various weights.